Image processing/Air_Hockey/main.py

```
@@ -0,0 +1,70 @@
+import cv2
+from utils.hand_detection import HandDetection
+from utils.Ball import Ball
+from utils.Paddle import Paddle
+from utils.collision import handle_collision
+from utils.constants import WIDTH, HEIGHT, LEFT_OFFSET, RIGHT_OFFSET
+from utils.Score import Score
```

Pylint (3.3.7)

[error] Cannot import 'utils.Score' due to 'invalid syntax (utils.Score, line 2)'

(E0001)